Dockerfile

```diff
@@ -1,20 +1,17 @@
-FROM node:16-buster
+FROM node:lts AS build
```

Comment:
isn't it better to use a specific version? node:lts will at some point be a different version